ruby/plezi/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM ruby:2.6
+
+WORKDIR /usr/src/app
+
+COPY Gemfile config.ru ./
+
+RUN bundle install
+
+CMD iodine -w -2 -t 2
```

Comment:
The Ruby `iodine` server settings used here might be suboptimal for the environment.

Currently, `iodine` will run the same amount of workers as half the detected CPU cores (`-2`), using 2 threads per worker:


A single thread might work better when there's no database involved. Performance is always a matter of balancing resources, which I did in a very offhanded manner.

Reply:
I understand you're concern @boazsegev. 
However
+ the implementation will be reviewed (changed) @see https://github.com/the-benchmarker/web-frameworks/issues/1728
+ I prefer to have the same optimisation in **iodine** that in **puma**